scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -111,7 +110,6 @@ workflow GvsQuickstartIntegration {
                 vcf_files_column_name = wgs_vcf_files_column_name,
                 vcf_index_files_column_name = wgs_vcf_index_files_column_name,
                 sample_set_name = select_first([wgs_sample_set_name, "wgs_integration_sample_set"]),
-                use_classic_VQSR = true,
```

Comment:
why was this taken out?

Reply:
same as above, but with line 101
We were worried that it opened us up to additional user error and it made more sense to only have the info input once and then it could be used however made sense in the actual task